### DISCUSSION TITLE
Airbridge 추적을 막을 수가 있나요?

### DISCUSSION THREAD

**Discussion by MoSalahStays** [List-KR/community#56] [Q&A]
https://static.airbridge.io/sdk/latest/airbridge.min.js 에서 SDK를 불러와서 https://id.abr.ge/, https://sdk.airbridge.io/, https://core.airbridge.io/ 에 뿌려주는 방식이네요. 차단 리스트에 추가 가능할까요?

**Comment by piquark6046** [comment]
AdGuard 추적 보호 필터에 이미 추가되어 있습니다.

**Reply by MoSalahStays** [reply]
Adguard 필터를 찾아볼 생각을 못했었네요. 감사합니다.
